src/uu/wc/src/utf8/lib.rs

```diff
@@ -0,0 +1,186 @@
+mod lossy;
```

Comment:
license is missing
it would be nice also to credit @SimonSapin 
and to explain why it is in tree

Reply:
I added a LICENSE and credited @SimonSapin with an explanation of the background of this issue in a comment at the top of `utf8/mod.rs`